docs/guide/docker-builder-vm.md

```diff
@@ -36,6 +36,20 @@ docker_builder:
 !!! info "Example"
     For more examples please check how we use Docker Builder to build and publish Cirrus CI's Docker Images for [Android](https://github.com/cirruslabs/docker-images-android).
 
+### Multi-arch builds
+
+Docker Builder VM has QEMU pre-installed and is able to execute multi-arch builds via [`buildx`](https://docs.docker.com/buildx/working-with-buildx/).
+Add the following `setup_script` to enabled `buildx` and then use `docker buildx build` instead of the regular `docker build`:
```

Comment:
Possible typo:

```suggestion
Add the following `setup_script` to enable `buildx` and then use `docker buildx build` instead of the regular `docker build`:
```